src/jit/lsra.cpp

```diff
@@ -6480,7 +6480,6 @@ void LinearScan::unassignPhysReg(RegRecord* regRec, RefPosition* spillRefPositio
     {
         // This must have been a temporary copy reg, but we can't assert that because there
         // may have been intervening RefPositions that were not copyRegs.
-        regRec->assignedInterval = nullptr;
```

Comment:
Also, a comment would be useful here explaining why we know it has already been set to nullptr.

---
In reply to: [124342695](https://github.com/dotnet/coreclr/pull/12474#discussion_r124342695) [](ancestors = 124342695)